rest, users: Update configuration to version 35. (#709)
syncthing/docs: users/config.rst

    <folder id="default" label="Default Folder" path="/Users/jb/Sync/" type="sendreceive" rescanIntervalS="3600" fsWatcherEnabled="true" fsWatcherDelayS="10" ignorePerms="false" autoNormalize="true">
        <filesystemType>basic</filesystemType>

>         <device id="S7UKX27-GI7ZTXS-GC6RKUA-7AJGZ44-C6NAYEB-HSKTJQK-KJHU2NO-CWV7EQW" introducedBy="">

Reviewer: Is there any particular reason for changing the existing device IDs (and API keys too, see below)?

Author: Yes.  I don't have access to the device certificate originally used to generate the examples.  So it generated a new certificate locally and that one's device ID always got added as a "new" (local) device everywhere.  To avoid fussing around each time I regenerate the XML, I just decided to switch over to the new ID.

Nobody will care I suppose, it's just an example for the encoded format.

        <disableFsync>false</disableFsync>
        <blockPullOrder>standard</blockPullOrder>
        <copyRangeMethod>standard</copyRangeMethod>
        <caseSensitiveFS>false</caseSensitiveFS>

>         <junctionsAsDirs>true</junctionsAsDirs>

Reviewer: ```suggestion
        <junctionsAsDirs>false</junctionsAsDirs>
```

Sticking to the convention of using default values here, shouldn't this say `false`? The default for `junctionsAsDirs` is `false`.

Author: Done.  I have no clue how it ended up there, because I just migrated the existing config.xml content. from the example.

        <device id="" compression="metadata" introducer="false" skipIntroductionRemovals="false" introducedBy="">
            <address>dynamic</address>
            <paused>false</paused>
            <autoAcceptFolders>false</autoAcceptFolders>
            <maxSendKbps>0</maxSendKbps>
            <maxRecvKbps>0</maxRecvKbps>
            <maxRequestKiB>0</maxRequestKiB>
            <untrusted>false</untrusted>
            <remoteGUIPort>0</remoteGUIPort>
        </device>
    </defaults>

The ``defaults`` element describes a template for newly added device and folder
options.  These will be used when adding a new remote device or folder, either
through the GUI or the command line interface.  The following child elements can
be present in the ``defaults`` element:

device
    Template for a ``device`` element, with the same internal structure.  Any
    fields here will be used for a newly added remote device.  The ``id``
    attribute is meaningless in this context.

folder
    Template for a ``folder`` element, with the same internal structure.  Any
    fields here will be used for a newly added shared folder.  The ``id``
    attribute is meaningless in this context.

>     The UI will propose to create new folders at the path given by the ``path``
>     attribute (used to be ``defaultFolderPath`` under ``options``).  It also
>     applies for automatically accepted folders from a remote device.

Reviewer: ```suggestion
    The UI will propose to create new folders at the path given in the ``path``
    attribute (used to be ``defaultFolderPath`` under ``options``).  It also
    applies to folders automatically accepted from a remote device.
```

Author: Applied, thanks.
